command.go

```diff
@@ -10,6 +10,13 @@ import (
 	"bitbucket.org/creachadair/shell"
 )
 
+type BashOpt string
+
+// StrictBashOpts contains two bash options 'pipefail' and 'errexit'. 'pipefail' instructs bash to fail an entire statement if any command in a pipefails. 
+// 'errexit' lets bash exit with an err exit code if a command fails . For more options see
+// 'man bash'
+var StrictBashOpts = []BashOpt{"pipefail", "errexit"}
```

Comment:
```suggestion
// BashOpt denotes options for running bash commands. For more options see 'man bash'
type BashOpt string

const (
    // 'pipefail' instructs bash to fail an entire statement if any command in a pipefails. 
    BashOptPipeFail BashOpt = "pipefail"
    // 'errexit' lets bash exit with an err exit code if a command fails .
    BashOptErrExit  BashOpt = "errexit"
)

// StrictBashOpts contains options that effectively enforce safe execution of bash commands.
var StrictBashOpts = []BashOpt{BashOptPipeFail, BashOptErrExit}
```

(feel free to adjust docstring as you see fit!)

Reply:
Ah <3 like this a lot!